flict/impl.py

```diff
@@ -87,8 +87,12 @@ def verify(self):
                              "Missing argument to the verify command")
 
     def _read_compliance_report(self, report_file):
-        with open(report_file) as fp:
-            return json.load(fp)
+        on_error = "Internal error, report file has improper format, required: JSON"
+        try:
+            with open(report_file) as file_:
+                return json.load(file_)
+        except json.decoder.JSONDecodeError:
+            raise FlictError(ReturnCodes.RET_INTERNAL_ERROR, on_error)
```

Comment:
Could you add some unit tests please - just so we have this covered and avoiding future regressions

Reply:
sure,  I'll add them